internal/database/testdb/testdb.go

```diff
@@ -0,0 +1,48 @@
+// Package testdb is a testing helper package which initializes a new crdb database and runs migrations
```

Comment:
Be mindful that symbols like the one in this file will (probably) be included in the final binary when built, along with their dependencies. You'll either need a build tag for this or to put it in a testing file.